Repository/Api/EditableRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Resource\Repository\Api;
+
+use InvalidArgumentException;
+use Puli\Repository\Api\EditableRepository;
+use Puli\Repository\Api\UnsupportedLanguageException;
+
+/**
+ * Extends the Puli editable repository to implement the as-of-yet not
+ * implemented features.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+interface EditableRepository extends EditableRepository
```

Comment:
travis say it does not:

```
PHP Fatal error: Cannot declare class
Symfony\Cmf\Component\Resource\Repository\Api\EditableRepository because
the name is already in use in
/home/travis/build/symfony-cmf/resource/Repository/Api/EditableRepository.php
on line 24 ```